stacker/actions/build.py

```diff
@@ -257,18 +280,15 @@ def _get_dependencies(self):
 
     def pre_run(self, outline=False, dump=False, *args, **kwargs):
         """Any steps that need to be taken prior to running the action."""
-        pre_build = self.context.config.get("pre_build")
-        should_run_hooks = (
-            not outline and
-            not dump and
-            pre_build
+        hooks = self.context.config.get("pre_build")
+        handle_hooks(
+            "post_build",
```

Comment:
Shouldn't this be `pre_build`? I don't think this is a breaking change since it's only used in logging.